python/spark_sklearn/grid_search.py

```diff
@@ -42,6 +53,11 @@ class GridSearchCV(BaseSearchCV):
 
     fit_params : dict, optional
         Parameters to pass to the fit method.
+         
+         .. deprecated:: 0.19
```

Comment:
This deprecation warning is for a scikit-learn version.  I recommend we just remove the argument from spark-sklearn.